blocks/Modal/Modal.js

```
@@ -83,15 +84,23 @@ export class Modal extends Block {
         this.hide();
-        this.setAttribute('aria-modal', 'false');
       }
     });
+
+    this.modalManager.subscribe(ModalEvents.CLOSE, (/** @type {{ id: string }} */ { id }) => {
+      if (id === this.id) {
+        this.hide();
+      }
+    });
+
+    this.modalManager.subscribe(ModalEvents.CLOSE_ALL, ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)